suites/smokeTests/brainTests.js

```diff
@@ -0,0 +1,198 @@
+
+// @toBeAutomated
+Scenario('Login with Synapse first time @manual @dream', ifInteractive(
+  async () => {
+    const result = await interactive(`
+            Note: execute this test, if the user is logging in with Syanpse account for the first time.
+            1. Go to https://gen3qa.braincommons.org/
+            2. Login with BEAT-PD button (with correct credentials and incorrect credentials)
+            3. the user is redirected to Synapse site, click on 'Allow' which redirects to https://gen3qa.braincommons.org/
+            4. After being re-directed, there is a 'Terms of Use and Privacy Policy' pop-up on https://gen3qa.braincommons.org/ 
+            5. the user have to hit 'Agree' two times to get through to the data commons
+            6. Log out of Synapse account
+        `);
+    expect(result.didPass, result.details).to.be.true;
+  },
+));
+
+Scenario('Login with Synapse @manual @dream', ifInteractive(
+  async () => {
+    const result = await interactive(`
+            1. Go to https://gen3qa.braincommons.org/
+            2. Login with BEAT-PD button (with correct credentials and incorrect credentials)
+            3. the user is redirected to Synapse site, click on 'Allow' which redirects to https://gen3qa.braincommons.org/
+            4. After being re-directed to the https://gen3qa.braincommons.org/, the user is logged in with the Synapse login email id
+        `);
+    expect(result.didPass, result.details).to.be.true;
+  },
+));
+
+Scenario('Exploration Page @manual @core', ifInteractive(
+  async () => {
+    const result = await interactive(`
+            1. Navigate to Exploraton tab, the page loads with facet search tab on the left and statistic graphs, aggregated total of project and cases and project table
+            2. Click on Data/File tabs on the Exploration Page, see if there is a change on the graphs and project/file table
+            3. Make selection in the facet search options, see if there are any changes in the graphs and project/file table
+            4. Click on the 'DownloadData' and 'DownloadManifest' button, see if the selected cohort is downloaded 
+        `);
+    expect(result.didPass, result.details).to.be.true;
+  },
+));
+
+Scenario('Profile Page @manual @core', ifInteractive(
+  async () => {
+    const result = await interactive(`
+            1. Click on Profile Tab on the top bar
+            2. the user sees 'Create API key' button and table with resource permissions
+            3. user clicks on 'Create API key' button and pop-up with API key created for the user
+            4. user can download the 'credential.json' to carry out various other test execution against https://gen3qa.braincommons.org/
+            5. the user is able see the project access table with right permissions to the right projects
+        `);
+    expect(result.didPass, result.details).to.be.true;
+  },
+));
+
+Scenario('Install gen3-client @manual @core', ifInteractive(
```

Comment:
Didn't we agree to replace @core with @smoke ?
All scenarios should contain 3 annotations:
```
@manual @smoke @brain
```

Right?